modules/social_features/social_group/modules/social_group_invite/social_group_invite.module

```diff
@@ -110,25 +110,44 @@ function social_group_invite_preprocess_views_view(&$variables) {
   /** @var \Drupal\views\ViewExecutable $view */
   $view = &$variables['view'];
 
+  if (!\Drupal::moduleHandler()->moduleExists('social_group_invite')) {
```

Comment:
@diakivvasyl This condition doesn't make sense. It will never work since we are already in the `social_group_invite` module. It means code in this file will be already executed only when the module is enabled.